Cannot find module '../build/Release/lwip_image'

Just installed lwip in another project and it throw this error, as I can understand the build fails.
Any ideas?

```
 → node index.js
module.js:341
    throw err;
    ^

Error: Cannot find module '../build/Release/lwip_image'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at /Users/nv/Projects/lwip-test/node_modules/lwip/lib/Image.js:3:22
    at Object.<anonymous> (/Users/nv/Projects/lwip-test/node_modules/lwip/lib/Image.js:16:3)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
```

node v5.6.0
npm 3.6.0
lwip 0.8.0

--- a/lib/Image.js
+++ b/lib/Image.js
@@ -1,6 +1,6 @@
 (function(undefined) {
 
-    var lwip_image = require('../build/Release/lwip_image');
+    var lwip_image = require('bindings')('lwip_image');
 
     function Image(pixelsBuf, width, height, trans, metadata) {
         this.__lwip = new lwip_image.LwipImage(pixelsBuf, width, height);

--- a/lib/ImagePrototypeInit.js
+++ b/lib/ImagePrototypeInit.js
@@ -6,8 +6,8 @@
         decree = require('decree'),
         defs = require('./defs'),
         util = require('./util'),
-        encoder = require('../build/Release/lwip_encoder'),
-        lwip_image = require('../build/Release/lwip_image'),
+        encoder = require('bindings')('lwip_encoder'),
+        lwip_image = require('bindings')('lwip_image'),
         normalizeColor = util.normalizeColor;
 
     var judges = {

--- a/lib/obtain.js
+++ b/lib/obtain.js
@@ -5,7 +5,7 @@
         decree = require('decree'),
         defs = require('./defs'),
         util = require('./util'),
-        decoder = require('../build/Release/lwip_decoder'),
+        decoder = require('bindings')('lwip_decoder'),
         Image = require('./Image');
 
     var openers = [{

--- a/package.json
+++ b/package.json
@@ -4,6 +4,7 @@
   "main": "index.js",
   "dependencies": {
     "async": "~0.9.0",
+    "bindings": "^1.2.1",
     "decree": "0.0.6",
     "nan": "~2.0.9"
   },

